packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -168,9 +188,15 @@ export class AzureCommunicationChatAdapter implements ChatAdapter {
     this.context.offStateChange(handler);
   }
 
-  async sendMessage(content: string): Promise<void> {
+  async sendMessage(content: string, options: SendMessageOptions = {}): Promise<void> {
     await this.asyncTeeErrorToEventEmitter(async () => {
-      await this.handlers.onSendMessage(content);
+      /* @conditional-compile-remove-from(stable): FILE_SHARING */
+      options.metadata = { ...options.metadata, ...this.fileSharingAdapter.getFileSharingMetadata() };
```

Comment:
See my comment above - `getFileSharingMetadata` is not part of the `FileUploadAdapter` public interface, but is used here.

Reply:
Yes, I missed it. Good catch.